WcaOnRails/lib/database_dumper.rb

```diff
@@ -570,15 +570,9 @@ def self.actions_to_column_sanitizers(columns_by_action)
     }.freeze,
     "vote_options" => :skip_all_rows,
     "votes" => :skip_all_rows,
-    "linkings" => {
-      where_clause: "",
-      column_sanitizers: actions_to_column_sanitizers(
-        copy: %w(
-          wca_id
-          wca_ids
-        ),
-      ),
-    }.freeze,
+    # The `linkings` table is a big one and not really important one,
+    # so we avoid copying all the rows. It has even led to mysql full table error.
```

Comment:
uber nit: This comes across a bit awkwardly. I'd rephrase it like this:

> We have seen MySQL full table errors when trying to copy the entire `linkings` table. Fortunately, it is a big and not really important table, so we can simply skip all its rows.

Reply:
Sounds way better, thanks! There is a funny part though :smiley: 
> Fortunately, it is a big [...] table

Reply:
Oh whoops! That's not fortunate =) How about simply: "Fortunately, it is a not really important table..."

Reply:
Yep, that's what I did!